academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,170 @@
+* Connection semantics - [ICS 3](https://github.com/cosmos/ibc/tree/master/spec/core/ics-003-connection-semantics)
+  Connections encapsulate two `ConnectionEnd` objects on two separate blockchains. Each `ConnectionEnd` is associated with a light client of the other blockchain - for example, the counterparty blockchain. The connection handshake is responsible for verifying that the light clients on each chain are the correct ones for their respective counterparties. Connections, once established, are responsible for facilitating all cross-chain verifications of an IBC state. A connection can be associated with any number of channels. 
+  How is a connection established with a handshake?
+
+  ![Connection state](images/connectionstate.png)
+
+* Channel semantics - [ICS 4](https://github.com/cosmos/ibc/tree/master/spec/core/ics-004-channel-and-packet-semantics)
+  A module on one blockchain can communicate with other modules on other blockchains by sending, receiving, and acknowledging packets through channels that are uniquely identified by the (`channelID`, `portID`) tuple. Establishing a channel with a handshake: 
+
+  ![Channel state](images/channelsstate.png)
+
+* Port allocation - [ICS 5](https://github.com/cosmos/ibc/tree/master/spec/core/ics-005-port-allocation)
+  An IBC module can bind to any number of ports. Each port must be identified by a unique `portID`. Since IBC is designed to be secure with mutually distrusted modules operating on the same ledger, binding a port returns a dynamic object capability. To take action on a particular port (for example, an open channel with its portID), a module must provide the dynamic object capability to the IBC handler. This requirement prevents a malicious module from opening channels with ports it does not own. Thus, IBC modules are responsible for claiming the capability that is returned on `BindPort`.
+  
+  ![Dataflow between clients](images/dataflow.png)
+
+* Vector commitments -[ICS 23](https://github.com/cosmos/ibc/tree/master/spec/core/ics-023-vector-commitments)
+  The proof format that all implementations must be able to produce and verify.
+* Host requirements - [ICS 24](https://github.com/cosmos/ibc/tree/master/spec/core/ics-024-host-requirements)
+  The paths that all IBC implementations must use to commit IBC messages.
```

Comment:
More than just the paths I'd say. From the spec itself:
"The minimal set of interfaces which must be provided and properties which must be fulfilled by a state machine hosting an implementation of the interblockchain communication protocol."